# Chapter 10 - DNA Structure and replication
## Structure of DNA
* 3' end is where the OH is, 5' is where the Phosphate is
* New nucleotides are attached to the 3' end
* Deoxyribose sugar has one oxygen atom in the ring, 4 carbon ring, 5th above 
* C, T, U: Single ring. A, G: Double ring
* A = T, C â‰¡ G

## DNA Replication
* Origin of Replication: Sites for binding of proteins (mainly initiation/auziliary)/AT-rich region.
* Bacteria with circular DNA: One origin of replication: initiator attached, then two helicases replace is and they move in opposite directions. Eukaryotes: multiple replication origins. See [this](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6742236/) and [this](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/origin-of-replication#:~:text=DNA%20replication%20origins%20are%20characterized,involved%20in%20regulating%20initiation%20events.)   
* Helicase: Forks and unwinds the two DNA strands
* Replication Fork: Point at which the two strands split
* DNA Polymerase III: Reads and creates a new copy in the 3' to 5' direction. New nucleotides are attached by removing OH at the 3' end and attaching the phosphate of the new nucleotide. (initially, 3 phosphates would be attached to the nucleotide, the other 2 are lost)
* Leading strand: From replication fork: 3' to 5' (creates 5' to 3' which is what you call the direction to be). Can be continuously copied by the Polymerase.
* Lagging strand: From replication fork: 5' to 3' (creates 3' to 5'). Polymerase reads in the reverse direction, so:
    * Primase: Adds an RNA primer to the strand, from where the DNA polymerase moves toward the replication fork creating an Okazaki fragment. Then primase again adds an RNA primer at a certain distance.
    * The RNA primers are converted to DNA by DNA polymerase I
    * Gaps between the former primers and Okazaki fragments (where there are still free OHs towards the 5' end) are bridged by the DNA ligase.
    * Optimal DNA ligase pH = 7.4

> [!WARNING]
> Normal direction of operation of these enzymes: 5' to 3'
> Direction of DNA ligase/DNA polymerase I is from 5' to 3' (of the new strand, so it appears to go opposite of the DNA polymerase III)

> [!TIP]
> Check out the following for DNA replication:
> https://www.labxchange.org/library/pathway/lx-pathway:ad7fbf7e-9fee-4989-b8c6-e5737d21cc91/items/lb:LabXchange:a21a9b48:lx_simulation:1/55881?fullscreen=true
> https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/origin-of-replication#:~:text=DNA%20replication%20origins%20are%20characterized,involved%20in%20regulating%20initiation%20events.
> https://www.youtube.com/watch?v=bee6PWUgPo8

> [!NOTE]
> Reverse Transcriptase: RNA to DNA: Used by viruses to insert their genetic code
> Nuclease: Degrade nucleic acids

## Nucleotides to Amino acids
* Transcription: DNA -> RNA: Nucleus
* Translation: RNA -> Polypeptide
* 20 amino acids, triplet codon for each amino acid: 64 total possibilities: 61 - amino acids, 3 - stop codons
* Three base codons in DNA -> complementary codons in RNA
* mRNA: carry coding sequence for protein synthesis. Genetic code table given is based on mRNA sequence.
* tRNA: Carry amino acid to ribosome during protein synthesis
* AUG: Methionine/Start codon. Proteins may be started with Methionine

### Transcription
* Promoter: At the beginning of a gene, RNA polymerase attaches here
* RNA synthesis continues till a terminator sequence is reached
* Inside the RNA polymerase, the two strands are split, the RNA nucleotides are matched with the nucleotides in one strand, and released from the DNA. DNA is then wound back towards the exit.

## Processing of Eukaryotic DNA
* In prokaryotes, the RNA produced immediately functions as mRNA.
* In eukaryotes:
  * RNA is produced within Nucleus
  * Processing is also done within the nucleus which is as follows:
  * Addition of cap and tail to protect from enzymes and help ribosomes recognize it.
  * Non-coding introns are removed and exons are glued together. Exons are the genes that are expressed. (RNA splicing) This helps in one sequence leading to multiple expressions. 

> [!NOTE]
> Define phenotype and genotype
